src/App.tsx

```diff
@@ -1,61 +1,103 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [option, setOption] = useState(0);
+  const [title, setTitle] = useState('');
+  const [initialTodos, setNewTodo] = useState(todosFromServer
+    .map(todo => ({ ...todo, user: getUser(todo.userId) })));
```

Comment:
```suggestion
  const [todos, setTodos] = useState(initialTodos);
```
it will be more understandable naming